data/json/itemgroups/magazines.json

```diff
@@ -94,6 +94,9 @@
     "id" : "mags_pistol_common",
     "//" : "Factory specification pistol magazines commonly owned by citizens.",
     "items": [
+      ["glock17_17", 90],
```

Comment:
This makes various glock 17 magazines 2x as common as all other magazine types combined, and glock magazines and makes glock magazines 4x as common as all other magazine types combined, take that 50 for the glockmag and divide it between all the glock magazines.  Also you seem to have removed the glock17_33 definition, so this will break, test your shit.

Reply:
I haven't had time to fix this KG, I knew it broke the shit. It was just a matter of I didn't have time to fix it.

Reply:
Also, Dropped the 17 round magazine spawn chance to 35% and the 22 round magazine spawn chance to 15%.

Reply:
> take that 50 for the glockmag and divide it between all the glock magazines.

Reply:
Is it K now tho?

Reply:
Nevermind, I just did what you asked...